opnsense/data_firewall_alias.go

```diff
@@ -0,0 +1,85 @@
+package opnsense
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/kradalby/opnsense-go/opnsense"
+	"github.com/satori/go.uuid"
+	"log"
+)
+
+func dataFirewallAlias() *schema.Resource {
+	return &schema.Resource{
+		Read: dataFirewallAliasRead,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+			},
+			"enabled": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+			"type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"content": {
+				Type: schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Computed: true,
+			},
+			//TODO add other fields
+		},
+	}
+}
+
+// Read will fetch the data of a resource.
+func dataFirewallAliasRead(d *schema.ResourceData, meta interface{}) error {
+	log.Printf("[TRACE] Getting OPNsense client from meta")
+	c := meta.(*opnsense.Client)
+	wanted_name := d.Get("name")
+
+	// list all alias
+	alias_list, err_list := c.AliasGetList()
+	if err_list != nil {
+		// temporary fix for the internal error API when we try to get an unreferenced UIID
+		if err_list.Error() == "Internal Error status code received" {
+			d.SetId("")
+			return nil
+		}
+		log.Printf("ERROR: \n%#v", err_list)
```

Comment:
Is this a debug output?, can it be made to follow the consistency of the rest of the errors?